src/plugins/multifile/multifile.c

```diff
@@ -903,7 +907,7 @@ int elektraMultifileError (Plugin * handle ELEKTRA_UNUSED, KeySet * returned ELE
 		keySetString (parentKey, s->fullPath);
 		if (resolver->kdbError)
 		{
-			resolver->kdbError (handle, returned, parentKey);
+			resolver->kdbError (resolver, returned, parentKey);
```

Comment:
It's nice how something like this goes undetected for three years and multiple releases.